Gopkg.toml

```diff
@@ -37,7 +37,7 @@ required = [
 
 [[override]]
   name = "knative.dev/pkg"
-  branch = "master"
+  branch = "release-0.8"
```

Comment:
Just so I understand, do we expect to switch this back after the 0.8 cut next week?